catalyst/contrib/nn/criterion/triplet.py

```diff
@@ -1,9 +1,13 @@
 # flake8: noqa
 # @TODO: code formatting issue for 20.07 release
```

Comment:
so, looks like it time to remove this barge :)

Reply:
done